File: Library App/LibraryTrackerApp.java
import java.util.Scanner;
import java.util.ArrayList;

public class LibraryTrackerApp {
    public static ArrayList<String> availableBooks = new ArrayList<>();
    public static ArrayList<String> borrowedBooks = new ArrayList<>();

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int choice;

        availableBooks.add("Things fall Apart");
        availableBooks.add("Son of the Soil");
        availableBooks.add("Last Days at Forcados High");
        availableBooks.add("Purple Hibiscus");
        availableBooks.add("Face Me I Face You");

        System.out.println("--- Welcome to Library Tracker ---");

        do {
            System.out.println("\n1. View all books");
            System.out.println("2. Borrow a book");
            System.out.println("3. Return a book");
            System.out.println("4. Exit");
            System.out.print("Select an option: ");
            choice = input.nextInt();
            input.nextLine(); 

            switch (choice) {
                case 1:
                    viewAllBooks();
                    break;

                case 2:
                    System.out.println("What book would you like to borrow?");
                    String borrowBook = input.nextLine();

                    for (int i = 0; i < availableBooks.size(); i++) {
                        if (availableBooks.get(i).equalsIgnoreCase(borrowBook)) {
                            System.out.println(availableBooks.get(i) + " has been borrowed.\n");
                            borrowedBooks.add(availableBooks.get(i));
                            availableBooks.remove(i);
                            break;
                        } else if (i == availableBooks.size() - 1) {
                            System.out.println("Book not found in library.\n");
                        }
                    }
                    break;

                case 3:
                    System.out.println("What book would you like to return?");
                    String returnBook = input.nextLine();

                    for (int i = 0; i < borrowedBooks.size(); i++) {
                        if (borrowedBooks.get(i).equalsIgnoreCase(returnBook)) {
                            System.out.println(borrowedBooks.get(i) + " has been returned.\n");
                            availableBooks.add(borrowedBooks.get(i));
                            borrowedBooks.remove(i);
                            break;
                        } else if (i == borrowedBooks.size() - 1) {
                            System.out.println("This book was not borrowed.\n");
                        }
                    }
                    break;

                case 4:
                    System.out.println("Exiting... Goodbye!");
                    break;

                default:
                    System.out.println("Invalid option. Try again.");
            }

        } while (choice != 4);

        input.close();
    }

    public static void viewAllBooks() {
        if (availableBooks.isEmpty()) {
            System.out.println("No books available right now.\n");
        } else {
            System.out.println("Available books are:");
            for (String book : availableBooks) {
                System.out.println("- " + book);
            }
            System.out.println();
        }
    }
}

File: Library App/library_tracker_app.py
available_books = [
    "Things fall Apart",
    "Son of the Soil",
    "Last Days at Forcados High",
    "Purple Hibiscus",
    "Face Me I Face You"
]

borrowed_books = []

def view_all_books():
    if not available_books:
        print("No books available right now.\n")
    else:
        print("Available books are:")
        for book in available_books:
            print("- " + book)
        print()

print("--- Welcome to Library Tracker ---")

while True:
    print("\n1. View all books")
    print("2. Borrow a book")
    print("3. Return a book")
    print("4. Exit")
    choice = input("Select an option: ")

    if choice == "1":
        view_all_books()

    elif choice == "2":
        borrow_book = input("What book would you like to borrow? ")
        for i in range(len(available_books)):
            if available_books[i].lower() == borrow_book.lower():
                print(available_books[i], "has been borrowed.\n")
                borrowed_books.append(available_books[i])
                available_books.pop(i)
                break
            elif i == len(available_books) - 1:
                print("Book not found in library.\n")

    elif choice == "3":
        return_book = input("What book would you like to return? ")
        for i in range(len(borrowed_books)):
            if borrowed_books[i].lower() == return_book.lower():
                print(borrowed_books[i], "has been returned.\n")
                available_books.append(borrowed_books[i])
                borrowed_books.pop(i)
                break
            elif i == len(borrowed_books) - 1:
                print("This book was not borrowed.\n")

    elif choice == "4":
        print("Exiting... Goodbye!")
        break

    else:
        print("Invalid option. Try again.")
